src/styles/presentation.scss

```
@@ -0,0 +1,36 @@
+.presentation {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 50px 8.5vw;
+  justify-content: center;
+  &__item {
+    display: block;
+    flex-basis: 25%;
+    color: #7c7c7c;
+    text-align: center;
+    svg {
+      height: 45px;
+    }
+  }
+  &__title {
+    padding: 48px 0 50px;
+    font-family: "Din", sans-serif;
```

```suggestion
    font-family: Din, sans-serif;
```
тут уже "" не дуже потрібні)